charts/acp/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: acp
 description: Cloudentity Authorization Control Plane
 type: application
-version: 2.2.1-6
```

Comment:
Not sure which version should I use here. We'd like to release 2.2.1 but here it's already defined `2.2.1-6`? Any ideas?